_sass/_imla-nama.scss

```diff
@@ -44,8 +44,8 @@ html {
 
 @counter-style urdu-abjad {
     system: fixed;
-    symbols: '\0627 ' '\628' '\62C' '\62F' '\6C1' '\648' '\632' '\62D' '\637' '\6CC' '\6A9' '\644' '\645' '\646' '\636' '\639' '\641' '\635' '\642' '\631' '\633' '\62A' '\62B' '\62E' '\630' '\638' '\63A' '\634';
-    /* symbols: 'ا' 'ب' 'ج' 'د' 'ه' 'و' 'ز' 'ح' 'ط' 'ی' 'ك' 'ل' 'م' 'ن' 'ص' 'ع' 'ف' 'ض' 'ق' 'ر' 'س' 'ت' 'ث' 'خ' 'ذ' 'ظ' 'غ' 'ش' */;
+    symbols: '\627 ' '\628' '\62C' '\62F' '\6C1' '\648' '\632' '\62D' '\637' '\6CC' '\6A9' '\644' '\645' '\646' '\633' '\639' '\641' '\635' '\642' '\631' '\634' '\62A' '\62B' '\62E' '\630' '\636' '\638' '\63A';
+    /* symbols: 'ا' 'ب' 'ج' 'د' 'ه' 'و' 'ز' 'ح' 'ط' 'ی' 'ك' 'ل' 'م' 'ن' 'ل' 'م' 'ن' 'س' 'ع' 'ف' 'ص' 'ق' 'ر' 'ش' 'ت' 'ث' 'خ' 'ذ' 'ض' 'ظ' 'غ' */;
     prefix: '(';
     suffix: ') ';
     } // custom counter style for urdu abjads. prefix and suffix brackets not working for some reason
```

Comment:
`prefix` and `suffix` are not added when the `counter()` function is used — see the [note in the spec](https://drafts.csswg.org/css-counter-styles/#counter-style-prefix). We can instead add the parentheses in the `content` property where we are using the counter, but all the other list counters are without parentheses anyway, so let's skip them.

Reply:
oh so that is why. removed in [8a3fd99](https://github.com/urduweb/imla/pull/17/commits/8a3fd99ec4760de7da693119e776bcb294d3c867).